gluster/block/glusterblock-provisioner.go

```diff
@@ -357,8 +385,12 @@ func parseClassParameters(params map[string]string, kubeclient kubernetes.Interf
 			if err != nil {
 				return nil, fmt.Errorf("glusterblock: failed to parse hacount %v ", k)
 			}
+			cfg.haCount = haCount
 		case "opmode":
 			parseOpmode = v
+		case "chapauth":
```

Comment:
@wongma7 I will send the doc update for everything in following PR. There are some good amount of stuff need to be updated.